src/state/zustand/swapperStore/utils.ts

```diff
@@ -1,8 +1,44 @@
 // Helper function to convert basis points to percentage
-import { bnOrZero } from 'lib/bignumber/bignumber'
+import type { AssetId } from '@shapeshiftoss/caip'
+import type { MarketData } from '@shapeshiftoss/types'
+import type { BigNumber } from 'lib/bignumber/bignumber'
+import { bn, bnOrZero, convertPrecision } from 'lib/bignumber/bignumber'
+import type { ProtocolFee } from 'lib/swapper/api'
 
 export const convertBasisPointsToDecimalPercentage = (basisPoints: string) =>
   bnOrZero(basisPoints).div(10000)
 
 export const convertBasisPointsToPercentage = (basisPoints: string) =>
   bnOrZero(basisPoints).div(100)
+
+type SumProtocolFeesToDenomArgs = {
+  cryptoMarketDataById: Partial<Record<AssetId, Pick<MarketData, 'price'>>>
+  outputAssetPriceUsd: string
+  outputExponent: number
+  protocolFees: Record<AssetId, ProtocolFee>
+}
+
+export const sumProtocolFeesToDenom = ({
```

Comment:
Can we add a smolish comment on top of this guy explaining that this converts in a `value` for a given asset? We usually refer to denom as the asset/symbol, not the "value in a certain denom" if that makes sense

Reply:
done